pages/ar/developer/assemblyscript-migration-guide.mdx

```diff
@@ -91,19 +91,19 @@ maybeValue.aMethod();
 ولكن في الإصدار الأحدث ، نظرًا لأن القيمة nullable ، فإنها تتطلب منك التحقق ، مثل هذا:
 
 ```typescript
-let maybeValue = load()
+let maybeValue = load();
```

Comment:
As mentioned on Slack, we should fix the translations on Crowdin instead of in this PR. I will take care of it when this PR is merged, but can you please revert your changes to non-English files?

Reply:
Done.